server/pkg/provision/auth.go

```diff
@@ -43,3 +61,119 @@ func emailSettings(ctx context.Context, s store.Storer) error {
 		})
 	})
 }
+
+// addAuthSuperUsers assigns BYPASS roles to users from AUTH_PROVISION_SUPER_USER value
+// When in Production, Corteza should stop and report an error.
+func addAuthSuperUsers(ctx context.Context, log *zap.Logger, s store.Storer, envOpt options.EnvironmentOpt) (err error) {
+	var (
+		val, has = os.LookupEnv("AUTH_PROVISION_SUPER_USER")
+	)
+
+	if envOpt.IsProduction() {
+		panic(errors.New(0, "Cannot provision super users in Production enviroment!"))
+	}
+
+	if has {
+		users := strings.Split(val, ";")
+
+		for _, user := range users {
+			// skip existing email
+			_, err = s.LookupUserByEmail(ctx, user)
+			if err != store.ErrNotFound {
+				log.Warn(fmt.Sprintf("Email address already %s exists", user))
+				continue
+			}
+
+			u := newUser(user)
+
+			createUserHandle(u)
```

Comment:
Why is this not part of the `newUser` fnc? Should this instead return the handle value?